tests/typings/sync/sync.ts

```diff
@@ -49,6 +49,27 @@ const executeResult = browser.execute(function (x: number) {
 }, 4)
 executeResult.toFixed(2)
 
+const executeAsyncResult = browser.executeAsync(function(a: any, b: any, c: any, d: any, done: any) {
+    setTimeout(() => {
+        done(a + b + c + d)
+    }, 100);
+}, 1, 2, 3, 4)
+executeAsyncResult.toEqual(10);
+
+const executeAsyncResultTwo = browser.executeAsync(function(done: any) {
+    setTimeout(() => {
+        done(5)
+    }, 100);
+})
+executeAsyncResultTwo.toEqual(5);
+
+const executeAsyncResultThree = browser.executeAsync(function(done: any) {
+    setTimeout(() => {
+        done('worked')
+    }, 100);
+})
+executeAsyncResultThree.toEqual('worked');
```

Comment:
where does `toEqual` come from?

Reply:
we can just call a primitive function to check that the return type is valid, e.g. for numbers `toFixed` or for strings `charCodeAt`